From 0396ac65a230af634f4937aa4d7bf3a79c5f6cfb Mon Sep 17 00:00:00 2001
From: Igal Liberman <igall@marvell.com>
Date: Tue, 26 Jun 2018 16:29:31 +0300
Subject: [PATCH 0757/1200] mvebu: config: use different env addresses

The ATF RT services region and TEE overlaps with existing env addresses.
Because the ATF RT service and TEE region is going to be marked as
secure and can't be overwritten, use different different addresses for
some envs.

Change-Id: I339592a0d068080ee218f8a4c7dd075ada3d3c35
Signed-off-by: Igal Liberman <igall@marvell.com>
Reviewed-on: http://vgitil04.il.marvell.com:8080/57362
Tested-by: iSoC Platform CI <ykjenk@marvell.com>
---
 include/configs/mvebu_armada-common.h | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/configs/mvebu_armada-common.h b/include/configs/mvebu_armada-common.h
index 87bcfc716c..b5194cd1f7 100644
--- a/include/configs/mvebu_armada-common.h
+++ b/include/configs/mvebu_armada-common.h
@@ -33,11 +33,11 @@
 #define CONFIG_HAS_ETH2
 #define CONFIG_ETHPRIME			"eth0"
 #define CONFIG_ROOTPATH                 "/srv/nfs/" /* Default Dir for NFS */
-#define CONFIG_EXTRA_ENV_SETTINGS	"kernel_addr_r=0x5000000\0"	\
+#define CONFIG_EXTRA_ENV_SETTINGS	"kernel_addr_r=0x7000000\0"	\
 					"initrd_addr=0xa00000\0"	\
 					"initrd_size=0x2000000\0"	\
-					"fdt_addr_r=0x4f00000\0"	\
-					"loadaddr=0x5000000\0"		\
+					"fdt_addr_r=0x6f00000\0"	\
+					"loadaddr=0x6000000\0"		\
 					"fdt_high=0xffffffffffffffff\0"	\
 					"hostname=marvell\0"		\
 					"ramdisk_addr_r=0x8000000\0"	\
-- 
2.22.0

